scripts/benchall.py

```
@@ -33,14 +33,14 @@ def get_target_from_name(name):
 
 
 benchmarks = [  # benchmark object, arguments
-        (clp_s_bench, {}),
-        (clickhouse_native_json_bench, {
+        (clp_bench, {}),
+        (clickhouse_bench, {
             'manual_column_names': False,
             'keys': [],
             'additional_order_by': [],
             'timestamp_key': True
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)